src/lib.rs

```diff
@@ -1180,6 +1201,7 @@ impl<T: Counter> Histogram<T> {
         let bucket_index = self.bucket_for(value);
         let sub_bucket_index = self.sub_bucket_for(value, bucket_index);
         // calculate distance to next value
+        // TODO when is sub_bucket_index >= sub_bucket_count?
```

Comment:
I'm actually not sure when this would be triggered. If I remember correctly, this is also present in the HdrHistogram code, and I just ported if over directly for fear of breaking something...